Book-A-Desk.Api.Tests/BookingsTests.fs

```diff
@@ -39,12 +39,16 @@ let mockReservationCommandFactory : ReservationCommandsFactory =
 
 [<Fact>]
 let ``GIVEN A Book-A-Desk server, WHEN booking a desk, THEN a desk is booked`` () = async {
-    let mockApiDependencyFactory = ApiDependencyFactory.provide mockProvideEventStore mockReservationCommandFactory mockGetOffices
+    let mutable emailWasSent = false
+    let mockEmailNotification booking =
+        emailWasSent <- true
+        ()    
+    let mockApiDependencyFactory = ApiDependencyFactory.provide mockProvideEventStore mockReservationCommandFactory mockGetOffices mockEmailNotification
     use httpClient = TestServer.createAndRun mockApiDependencyFactory
 
     let booking  =
         {
-            Office = { Id = mockOfficeId.ToString() }
+            Office = { Id = mockOfficeId }
```

Comment:
Based on the comment below (about usage of domain models in API model this probably needs to be reverted or changed.